### ISSUE TITLE
Ability to add expense in a different currency

### ISSUE THREAD

**Issue by borlaym** [mvarnagiris/financius#332]
I'd like the ability to add expenses in currencies different than the default currency. Comes up often when shopping online. It should convert to the default currency using that day's exchange rates, I wouldn't even mind not keeping the price in the other currency later.

**Comment by denilsonsa** [comment]
Indeed, it happens often when shopping online and paying with the credit card.

However, give the actual exchange rate happens when the credit card bill is paid, the actual exchange rate will be different than the one from the day of the expense. So, yeah, both values should be kept (even if as plain text, appended to the note textarea).

**Comment by tkrunning** [comment]
Yes! This is so badly needed. Most of my transactions are of this kind (I'm traveling a lot, so my credit card currency is rarely the same as the local currency). I've been trying to use Financius for tracking my expenses while traveling, but the lack of this feature is why I eventually stopped using the app after a few months regular use. Now, a year later I am trying to get back into tracking, but I still find the lack of this feature to be a major pain.

Ideal workflow: select different currency when adding a transaction, add the local price, the app should then estimate the cost in the main currency for the account being charged. The transaction should then default to uncheck the confirmed option, so it's easy to spot transactions later that I still need to make a final adjustment to.
